swift/src/IceImpl/BlobjectFacade.h

```diff
@@ -27,25 +30,21 @@ ICEIMPL_API @protocol ICEBlobjectFacade
            requestId:(int32_t)requestId
        encodingMajor:(uint8_t)encodingMajor
        encodingMinor:(uint8_t)encodingMinor
-            response:(ICEBlobjectResponse)response
-           exception:(ICEBlobjectException)exception;
+        sendResponse:(ICESendResponse)sendResponse;
 - (void)facadeRemoved;
 @end
 
 #ifdef __cplusplus
 
-class BlobjectFacade : public Ice::BlobjectArrayAsync
+/// A C++ dispatcher that dispatches requests to a Swift object via an ObjC/Swift "facade".
+class SwiftDispatcher final : public Ice::Object
```

Comment:
Should this file be renamed if we're renaming the class.

Reply:
Yes, in a follow-up PR.